descriptor.md

```diff
@@ -0,0 +1,113 @@
+<!--[metadata]>
++++
+draft = true
++++
+<![end-metadata]-->
+
+# OpenContainers Content Descriptors
+
+OCI have several components that come to together to describe an image. 
+References between components form a [Merkle Directed Acyclic Graph (DAG)](https://en.wikipedia.org/wiki/Merkle_tree).
+The references in the _Merkle DAG_ are expressed through _Content Descriptors_.
+A _Content Descriptor_ or _Descriptor_, describes the disposition of targeted content.
+A _Descriptor_ includes the type of content, an independently-verifiable content identifier, known as a "digest" and the byte-size of the raw content.
+
+Descriptors SHOULD be embedded in other formats to securely reference external content.
+
+Descriptors MAY be used as an independent object to specify content.
+
+Implementations MUST verify fetched content against 
+
+## Properties
+
+- **`mediaType`** *string*
+
+  This REQUIRED property contains the MIME type of the referenced object.
+
+- **`digest`** *string*
+
+  This REQUIRED parameter is the digest of the content, meeting the requirements outlined in [Digests and Verification](#digests-and—verification).
+  Retrieved content SHOULD be verified against this digest before processing.
```

Comment:
This should probably say MUST.

> if you know a separate local layer has already verified the hash, there's no sense in having your layer do it again.

If you already have it locally, the requirement does not apply. I'll add a blurb about this in the section on calculation saying that it is okay to cache calculation.